src/createServer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const express = require('express');
+const cors = require('cors');
+
+const { usersService } = require('./services/users');
```

Comment:
It seems like you are working with categories, but the services and routers in this file are related to users and expenses. Please update the imports and usage to match the task requirements.